src/librustc_typeck/check/mod.rs

```diff
@@ -544,14 +544,20 @@ pub fn check_item_bodies<'a, 'tcx>(tcx: TyCtxt<'a, 'tcx, 'tcx>) -> CompileResult
     });
 
     fn check_item_bodies_task<'a, 'tcx>(tcx: TyCtxt<'a, 'tcx, 'tcx>, (): ()) {
-        tcx.visit_all_bodies_in_krate(|body_owner_def_id, _body_id| {
-            tcx.item_tables(body_owner_def_id);
-        });
+        ty::queries::typeck_item_bodies::get(tcx, DUMMY_SP, LOCAL_CRATE)
```

Comment:
For the moment, yes, because otherwise it causes and ICE with "read/write of `Some(TypeckBodiesKrate)` but no current task.

Should I try and fix this here somehow or just add a `FIXME`?

cc @nikomatsakis